### ISSUE TITLE
Ada not included when filtering Vendors for uncollected items

### ISSUE THREAD

**Issue by chrisfried** [DestinyItemManager/DIM#7023]
### DIM Version

Version 6.81.0.1000691 (beta), built on 9/8/2021, 9:04:55 AM

### Browser Details

Chrome 92.0.4515.159 (Official Build) (64-bit)

### OS Details

Windows 10

### Describe the bug

Ada is currently selling the Insight armor sets:
![image](https://user-images.githubusercontent.com/422381/132529410-b9f6ea9e-25c5-4586-a1ab-765399e1ad11.png)

I do not have this set in my collection:
![image](https://user-images.githubusercontent.com/422381/132529504-94240a1f-1b98-47d4-9d6f-f98ef02e09f3.png)

However, when I filter the Vendor page to "Only show uncollected items," the items do not appear, and Ada is filtered out of the list:
![image](https://user-images.githubusercontent.com/422381/132529716-26730446-cc0e-4dbe-bbeb-bd909da0aaa2.png)

**Comment by chrisfried** [comment]
In-game collection:
![image](https://user-images.githubusercontent.com/422381/132532013-92c97eaf-8fb5-4240-bfdf-c1a251b7d146.png)

**Comment by chrisfried** [comment]
In-game Ada:
![image](https://user-images.githubusercontent.com/422381/132535550-e3bbfd4c-f03d-4993-b69f-e8261104be89.png)

**Comment by nev-r** [comment]
confirmed these are completely separate items.
ada is selling https://data.destinysets.com/i/InventoryItem:2364041279

and the collectible
( https://data.destinysets.com/i/Collectible:2590296094 )
and the item's newer version
( https://data.destinysets.com/i/InventoryItem:2294753710 )
point to each other

so this is would be a matter of either programatically linking 2364041279 and 2294753710 as being synonyms, or declaring defeat

**Comment by chrisfried** [comment]
Ada's Vendor list in DIM after purchasing the hat and gloves, then dismantling the gloves.
![image](https://user-images.githubusercontent.com/422381/132537277-f031921e-7072-46e2-b4da-fb42685213b5.png)

**Comment by chrisfried** [comment]
Created a Bungie API issue: https://github.com/Bungie-net/api/issues/1523

**Comment by chrisfried** [comment]
Logged in to quickly pick up missing items before reset. Found a fun new twist on this bug:

Ada is selling the Warlock Ego Talon Suit. In this screenshot, I purchased the helmet:
![image](https://user-images.githubusercontent.com/422381/135117195-748ef36b-63c1-43aa-af94-f87d3d99ba6e.png)

I own one item in the above screenshot, but initially, I didn't own any. I went to my Collections to check if it was in there. It's an old, "unobtainable" set, so Collections didn't even show the header for the item set until I searched for it:
![image](https://user-images.githubusercontent.com/422381/135117505-0a1d8467-1abc-4733-aefb-e26c530bfbc5.png)

After obtaining the helmet, Collections doesn't show the missing pieces:
![image](https://user-images.githubusercontent.com/422381/135117606-ed55fbfc-1353-46f7-bb77-11d5aac0c2d0.png)

Arguably, if Ada is selling an item, it should appear in Collections as something that can be obtained. cc: https://github.com/Bungie-net/api/issues/1523
